docs/src/_getting-started/part4.md

```diff
@@ -0,0 +1,194 @@
+Repository:     https://github.com/meltano/files-dbt-postgres
+Documentation:  https://hub.meltano.com/files/files-dbt-postgres
+
+Installing transformer 'dbt-postgres'...
+---> 100%
+Installing file bundle 'files-dbt-postgres'..
+---> 100%
+Installed file bundle 'files-dbt-postgres'
+
+Adding 'files-dbt-postgres' files to project...
+Created transform/.gitignore (files-dbt-postgres)
+Created transform/dbt_project (files-dbt-postgres).yml
+Created transform/models/.gitkeep (files-dbt-postgres)
+Created transform/profiles/postgres/profiles (files-dbt-postgres).yml
+
+Installed transformer 'dbt-postgres'
+Installed 2/2 plugins
+
+To learn more about transformer 'dbt-postgres', visit https://docs.meltano.com/guide/transformation
+To learn more about file bundle 'files-dbt-postgres', visit https://hub.meltano.com/files/files-dbt-postgres
+```
+
+</div>
+
+<br />
+As you can see, this adds both the transformer as well as a "file bundle" to your project. You can verify that this worked by viewing the newly populated directory `transform`.
+
+## Configure dbt
+Configure the dbt-postgres transformer to use the same configuration as our target-postgres loader using `meltano config`:
+
+<div class="termy">
+```console
+$ meltano config dbt-postgres set host localhost
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'host' was set in `meltano.yml`: 'localhost'
+$ meltano config dbt-postgres set port 5432
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'port' was set in `meltano.yml`: 5432
+$ meltano config dbt-postgres set user meltano
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'user' was set in `meltano.yml`: 'meltano'
+$ meltano config dbt-postgres set password password
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'password' was set in `.env`: 'password'
+$ meltano config dbt-postgres set dbname postgres
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'dbname' was set in `meltano.yml`: 'postgres'
+$ meltano config dbt-postgres set schema analytics
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'schema' was set in `meltano.yml`: 'analytics'
+```
+</div>
+
+## Add our source data
+The E(t)L pipeline run already added our source data into the schema `tap_github` as table `commits`. So let's add that to dbt to work with:
+
+```bash
+mkdir transform/models/tap_github
+```
+
+Add a file called `source.yml` into this directory with the following content:
+
+```yaml
+config-version: 2
+version: 2
+sources:
+  - name: tap_github     # the name we want to reference this source by
+    schema: tap_github   # the schema the raw data was loaded into
+    tables:
+      - name: commits
+```
+
+## Add a transformed model
+Add a file called `authors.sql` to the folder `transform/models/tap_github` with the following contents:
+
+```sql
+{% raw %}
+{{
+  config(
+    materialized='table'
+  )
+}}
+
+
+with base as (select *
+from {{ source('tap_github', 'commits') }}) {% endraw %}
+
+select distinct (commit -> 'author' -> 'name') as authors from base
```

Comment:
```suggestion
WITH base AS (

  SELECT *
  FROM {{ source('tap_github', 'commits') }}
  
) {% endraw %}

SELECT 
  DISTINCT (commit -> 'author' -> 'name') AS authors 
FROM base
```

I'm going to be annoying about structuring SQL https://handbook.meltano.com/data-team/sql-style-guide 😈